backend/model/EventParticipant.js

```diff
@@ -13,10 +13,18 @@ export function EventParticipantModel(sequelize) {
             defaultValue: sql.uuidV4,
             primaryKey: true,
         },
-        event_time_preference: {
-            type: DataTypes.DATE,
+        event_id: {
+            type: DataTypes.UUID.V4,
+            allowNull: false,
+        },
+        user_id: {
+            type: DataTypes.UUID.V4,
             allowNull: false,
-        }
+        },
+        accepted: {
+            type: DataTypes.BOOLEAN,
+            allowNull: true,
+        },
```

Comment:
when resolving merge conflicts, I think you should keep "status" instead of "accepted"